UnoCheck/Process/ShellProcessRunner.cs

```diff
@@ -23,11 +21,11 @@ public ShellProcessRunnerOptions(string exe, string args, CancellationToken canc
 			Args = args;
 			CancellationToken = cancellationToken;
 		}
-		 
+
 		public string Executable { get; set; }
 		public string Args { get; set; }
 
-		public bool Verbose { get;set; }
+		public bool Verbose { get; set; }
```

Comment:
Those changes look unrelated, make sure to remove them before removing the draft status.

Reply:
Done!